.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: python
+dist: bionic
+before_install:
+  - sudo apt-get update
+  - sudo apt-get -y install gnupg2 libgpgme-dev swig
+install:
+  - pip install --upgrade poetry
+  - poetry install
+matrix:
+  include:
+  - name: "Lint and static analysis"
+    python: 3.6
+    script:
+      - poetry run ./scripts/copyright_line_check.sh
+      - poetry run black --check --config pyproject.toml .
+      - poetry run flake8 --config setup.cfg . d
+      - poetry run isort --check --settings-path setup.cfg -rc .
```

Comment:
Consider adding `pylint` and `bandit` as well, possibly in a later PR.

Reply:
having some trouble getting pylint to work. I think I'll make an issue as a TODO to add pylint, bandit, and mypy.